cfg_VHcc_ZLL.py

```diff
@@ -122,21 +123,48 @@
         #"baseline_2L2J_ctag_calib": [passthrough],
         "presel_mumu_2J": [mumu_2j],
         "presel_ee_2J": [ee_2j],
-
+        
         "SR_mumu_2J_cJ": [Zmumu_2j, ctag_j1, dijet_mass_cut],
         "SR_ee_2J_cJ": [Zee_2j, ctag_j1, dijet_mass_cut],
         "SR_ll_2J_cJ": [Zll_2j, ctag_j1, dijet_mass_cut],
 
         "SR_ll_2J_cJ_low":  [Zll_2j, ctag_j1, dijet_mass_cut, dilep_pt60to150],
         "SR_ll_2J_cJ_high": [Zll_2j, ctag_j1, dijet_mass_cut, dilep_pt150to2000],
 
-
+        
 
         "CR_ll_2J_LF": [Zll_2j, antictag_j1, dijet_mass_cut],
         "CR_ll_2J_HF": [Zll_2j, btag_j1, dijet_mass_cut],
         "CR_ll_2J_CC": [Zll_2j, ctag_j1, dijet_invmass_cut],
         "CR_ll_4J_TT": [ll_antiZ_4j, btag_j1, dijet_mass_cut]
     },
+    
+    columns = {
+        "common": {
+            "bycategory": {
+                    "SR_ll_2J_cJ": [
+                        ColOut("events", ["EventNr", "dijet_m", "dijet_pt", "dijet_dr", "dijet_deltaPhi", "dijet_deltaEta",
+                                          "dilep_pt", "dilep_dr", "dilep_deltaPhi", "dilep_deltaEta",
+                                          "dijet_CvsL_max", "dijet_CvsL_min", "dijet_CvsB_max", "dijet_CvsB_min",
+                                          "dijet_pt_max", "dijet_pt_min", "ZH_pt_ratio", "ZH_deltaPhi", "deltaPhi_l2_j1", "deltaPhi_l2_j2"], flatten=False),
+                    ],
+                    "baseline_2L2J_no_ctag": [
+                        ColOut("events", ["EventNr", "dijet_m", "dijet_pt", "dijet_dr", "dijet_deltaPhi", "dijet_deltaEta",
+                                          "dilep_pt", "dilep_dr", "dilep_deltaPhi", "dilep_deltaEta",
+                                          "dijet_CvsL_max", "dijet_CvsL_min", "dijet_CvsB_max", "dijet_CvsB_min",
+                                          "dijet_pt_max", "dijet_pt_min", "ZH_pt_ratio", "ZH_deltaPhi", "deltaPhi_l2_j1", "deltaPhi_l2_j2"], flatten=False),
+                    ]
+                }
+        },
+        #"bysample":{
+        #    "ttHTobb":{
+        #        "bycategory": {
+        #            "semilep_LHE": [ColOut("HiggsParton",
+        #                                   ["pt","eta","phi","mass","pdgId"], pos_end=1, store_size=False, flatten=False)]
```

Comment:
I think this part is not needed - please remove.

Reply:
Removed